client/my-sites/stats/activity-log-confirm-dialog/index.jsx

```diff
@@ -8,98 +9,87 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import Dialog from 'components/dialog';
+import ActivityIcon from '../activity-log-item/activity-icon';
 import Button from 'components/button';
+import Card from 'components/card';
 import Gridicon from 'gridicons';
+import HappychatButton from 'components/happychat/button';
 
 class ActivityLogConfirmDialog extends Component {
 	static propTypes = {
 		applySiteOffset: PropTypes.func.isRequired,
-		isVisible: PropTypes.bool.isRequired,
 		onClose: PropTypes.func.isRequired,
 		onConfirm: PropTypes.func.isRequired,
-		siteTitle: PropTypes.string,
 		timestamp: PropTypes.number,
 
 		// Localize
 		translate: PropTypes.func.isRequired,
 		moment: PropTypes.func.isRequired,
 	};
 
-	static defaultProps = {
-		isVisible: false,
-	};
-
 	renderButtons() {
-		const {
-			onClose,
-			onConfirm,
-			translate,
-		} = this.props;
+		const { onClose, onConfirm, translate } = this.props;
 		return (
 			<div>
-				<Button onClick={ onClose }>
-					{ translate( 'Cancel' ) }
-				</Button>
-				<Button primary scary onClick={ onConfirm }>
-					{ translate( 'Restore' ) }
+				<a
+					className="activity-log-confirm-dialog__more-info-link"
+					href="https://help.vaultpress.com/one-click-restore/"
+				>
+					{ translate( '{{icon /}} More info', {
+						components: { icon: <Gridicon icon={ 'notice' } /> },
+					} ) }
+				</a>
+				<HappychatButton
+					className="activity-log-confirm-dialog__more-info-link"
+					href="https://help.vaultpress.com/one-click-restore/"
+				>
+					{ translate( '{{icon /}} Any Questions?', {
+						components: { icon: <Gridicon icon={ 'chat' } /> },
+					} ) }
+				</HappychatButton>
```

Comment:
Just noting this in relation to the work @nosolosw has been working on - are we wanting to intentionally further couple a project whose path is divergent from this one?

Reply:
I'm not sure if your question is directed at @nosolosw or me 🙂 

I'm aware that there is work to move HC to an extension, but I'm not aware of any changes in the way framework components like this should be used. I am happy to make the pertinent changes.

Reply:
Thanks for making me aware of this Dennis! I wouldn't want to be a stopping force, so go ahead with using the component. I'm focused now on the underlying tech, so it'll be a few weeks until I can move forward the Happychat extension (or separate npm package) work.